agent/cloudinit/cloudinit_integration_test.go

```diff
@@ -143,6 +156,13 @@ runCmd:
 		Expect(string(fileContents)).To(Equal(replacedFileContent))
 	})
 
+	It("should return error for invalid template content", func() {
+		cloudInitScript := "chcdslaxas;llc;"
```

Comment:
I understand the intention here, still can we change it to `"invalid-template"`, or `"foobar"`

Reply:
Yeahh makes it more readable 👍